winter_messaging_transactional/setup.py

```diff
@@ -0,0 +1,50 @@
+import importlib
+import inspect
+import os
+
+from sqlalchemy.engine import Engine
+
+from winter.core import get_injector
+from .injection_modules import TransactionalMessagingModule
+from .messaging_app import MessagingApp
+from .table_metadata import messaging_metadata
+
+
+class InvalidConfiguration(Exception):
+    pass
+
+
+def setup():
+    injector = get_injector()
+    messaging_app = _create_messaging_app()
+    messaging_app.setup(injector)
+    injector.binder.install(TransactionalMessagingModule)
+    engine = injector.get(Engine)
+    messaging_metadata.create_all(engine)
+
+
+def _create_messaging_app() -> MessagingApp:
+    winter_settings_module = _init_winter_settings()
+
+    messaging_app_class = None
+    for class_name, class_ in inspect.getmembers(winter_settings_module, inspect.isclass):
+        if issubclass(class_, MessagingApp) and class_ is not MessagingApp:
+            messaging_app_class = class_
+            break
+
+    if not messaging_app_class:
+        raise InvalidConfiguration('Define subclass of MessagingApp and override setup method')
+
+    return messaging_app_class()
+
+
+def _init_winter_settings():
+    winter_package_name = os.environ.get('WINTER_SETTINGS_MODULE')
```

Comment:
I didn't get what the variable is for. In general - let's add some basic info and guidelines to README.md on how to use the new messaging mechanism.